### DISCUSSION TITLE
it don t work for b1 lvl

### DISCUSSION THREAD

**Discussion by xXAloneX** [m1dugh/RosettaStonks#104] [General]
![image](https://github.com/user-attachments/assets/ddbb52e2-a960-4963-9357-8db995a9bdd8)

**Comment by np4abdou** [comment]
It works after 24h/72h with 30% chance *-*

**Reply by T1ttySl4yer** [reply]
it's been 1.5 weeks, it doesn't work a all.

**Reply by mohammed904** [reply]
Do you have any tricks to make it work?

**Reply by m1dugh** [reply]
It works instantly if you finish the lesson.

**Reply by mohammed904** [reply]
This works for English and after that won't work

**Reply by mohammed904** [reply]
Even wen I finish the lesson

**Reply by m1dugh** [reply]
@mohammed904 did it work ?

**Reply by mohammed904** [reply]
Yes but after 2 weeks
